src/js/media-chrome-listitem.js

```diff
@@ -10,6 +10,7 @@ template.innerHTML = `
     line-height: 1em;
     padding: 0.5em;
     margin: 0em;
+    cursor: pointer;
```

Comment:
thought(non-blocking): I'm always torn by this one, since I like the cursor to let me know something is clickable but I at least sometimes want to copy/paste some text that happens to be clickable.